docs/guides/manually-running.md

```diff
@@ -30,7 +30,7 @@ const { validationResult, ValidationChain } = require('express-validator');
 // sequential processing, stops running validations chain if the previous one fails.
 const validate = validations => {
   return async (req, res, next) => {
-    for (let validation of validations) {
+    for (const validation of validations) {
```

Comment:
Can you do the same to the TypeScript counterpart of this code block?

Reply:
Yup, sure.
Done ✅